framework/db/mssql/Schema.php

```diff
@@ -823,5 +823,4 @@ public function createColumnSchemaBuilder($type, $length = null)
     {
         return Yii::createObject(ColumnSchemaBuilder::className(), [$type, $length, $this->db]);
     }
-
```

Comment:
No need to change it.

Reply:
returned the codestyle to its original form